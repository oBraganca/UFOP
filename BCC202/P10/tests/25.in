25
-18 27 -6 12 -10 -2 9 -13 10 13 0 -4 -7 -5 -8 5 11 8 2 7 -12 1 -11 3 -3 4 6 -9 -1
0 45 15 -10 16 22 -3 -4 12 3 13 14 -14 -16 -2 19 17 4 -18 -11 -15 5 1 -22 -19 7 21 9 0 -9 11 2 -1 -7 -12 -8 18 -6 6 8 -21 -17 -13 20 -20 10 -5
15 28 -12 -13 2 -1 6 9 4 -8 -9 12 -3 8 -5 -14 10 1 -2 11 13 0 -4 7 5 -6 -11 -7 -10 3
-65 81 30 -29 36 -17 0 16 8 2 7 3 5 -31 -21 -40 18 23 4 19 -25 -26 -32 -38 20 -35 -3 26 25 22 -34 35 -8 -12 10 1 -23 12 -5 -6 -18 27 -1 28 37 17 -37 -33 38 -24 -4 34 39 32 -9 -27 -30 40 24 33 -13 11 9 -14 -15 -11 -2 21 -16 -36 31 -28 -10 -20 -39 15 6 -22 13 -7 14 -19 29
35 44 -14 -21 -20 -12 0 -18 9 1 -16 13 16 -5 10 5 -7 7 18 -4 3 15 -6 17 20 -2 -9 11 -3 12 -19 -15 -10 -13 4 6 -8 14 -11 21 19 -22 2 -17 -1 8
-62 62 -12 22 27 30 -9 -19 -5 -6 21 25 14 8 -20 12 7 16 -11 -18 -28 -8 23 -13 -4 -27 -29 9 -24 6 17 -14 3 11 13 -23 4 19 10 -7 -31 -30 15 -22 5 29 1 -17 24 -26 -1 -3 -10 0 20 26 -16 -2 28 -15 -21 18 -25 2
31 80 -1 7 28 34 -28 29 -12 -34 17 -38 -15 37 27 -21 -39 14 26 -7 -17 -5 18 -29 22 20 12 9 21 -16 0 -36 -40 24 10 35 25 -14 -20 -3 39 -33 33 6 -32 -23 -18 36 38 16 -26 30 -37 -31 -11 15 31 11 19 -9 4 -27 -6 -25 3 32 -24 -4 2 8 -30 -10 -35 5 -13 -22 -8 -2 1 13 23 -19
-26 35 3 7 -7 -11 11 -14 0 -2 -5 13 -9 9 -15 10 8 -16 -8 -12 -10 14 -13 2 -1 16 4 1 15 12 6 17 5 -4 -3 -17 -6
-32 98 31 -23 42 0 -48 18 38 -11 30 -28 19 -22 -16 -4 -1 6 -21 13 -42 -20 20 45 -47 9 16 -31 -8 11 -2 7 -36 37 21 22 48 -41 -27 14 4 23 -18 29 34 26 -19 15 -10 -5 -17 10 -37 -30 39 -25 27 -38 44 -44 -43 -40 -45 33 -49 17 -14 -3 1 40 -12 41 -26 -34 -33 32 47 24 5 -24 -13 2 -29 25 -39 36 43 -46 -9 -6 -15 8 3 46 -7 28 35 -32 12 -35
-19 92 40 37 24 -12 23 21 -2 36 -22 -7 44 -26 -21 7 -16 -36 -11 20 -41 -10 35 5 25 -37 6 -27 -9 38 30 -19 -8 -32 1 -28 13 29 -30 -3 -39 10 41 43 34 17 3 -24 -46 2 -40 -20 8 -18 -33 28 -43 9 -1 -23 0 -35 -15 -5 -25 31 42 -38 45 19 -4 18 14 -44 26 -14 -29 -45 -13 33 12 22 -34 27 -31 32 -42 -17 39 15 11 4 16 -6
3 57 10 4 -27 26 1 -10 -22 -4 24 9 -1 -19 -9 5 27 -6 -26 7 16 -28 13 12 -24 -5 -3 -13 25 15 -11 17 19 -20 -25 -8 2 -18 20 -7 8 14 11 18 21 6 -17 -21 0 23 -14 22 -16 -23 -2 -15 -12 28 3
0 53 -6 -5 -19 -16 -7 3 -11 8 17 0 7 21 -2 19 13 -25 -9 12 4 -17 16 24 -10 11 -26 1 -3 5 -12 -13 18 -4 20 14 2 15 -22 22 25 26 -23 10 -8 23 9 -20 -15 -24 -1 6 -14 -18 -21
-38 46 -15 -11 22 5 -20 11 12 14 -21 19 -8 -14 -12 -22 -7 6 10 -17 20 -5 -3 1 -10 0 21 -19 7 18 2 17 -6 -16 15 13 4 3 16 -1 9 -2 -4 -9 -18 -23 -13 8
-17 20 2 -9 8 0 6 -5 -10 3 7 1 9 4 -1 5 -3 -6 -8 -4 -2 -7
16 86 32 33 2 16 -2 -35 -4 37 21 -42 -29 0 5 -23 12 -32 -40 15 -10 -3 10 -36 -26 -24 42 6 -7 34 26 31 38 14 35 -30 40 22 11 -18 -1 29 27 -8 -39 -33 -28 41 17 -25 -19 -34 -14 20 1 -31 24 19 -16 -17 8 -22 -27 9 -38 -5 18 4 23 -21 -9 -11 -15 28 -13 3 30 36 -37 13 -12 -20 39 7 -41 25 -43 -6
42 51 -23 15 10 23 -5 -18 -13 -6 3 25 -1 22 13 -16 7 -14 1 -22 18 -17 -19 -15 2 -12 -7 -4 12 -2 11 8 -25 9 20 -24 -3 -21 6 -20 -8 5 4 19 -11 -9 16 24 0 14 21 -10 17
-28 49 11 -7 17 -12 24 -4 21 -13 -19 7 -24 -11 3 -2 14 -21 13 -10 9 0 18 -16 15 19 -6 -17 -8 -22 1 -23 -5 8 -3 10 -14 -18 -20 22 2 -9 20 4 -1 23 5 16 6 -15 12
-43 53 13 1 -16 -20 6 -10 -6 5 -13 -17 26 23 -4 21 7 -15 14 -9 24 -1 4 -5 -21 -19 -12 -7 10 25 -2 -11 15 -23 -18 -22 -3 18 3 2 22 -24 -26 20 16 8 -25 11 -14 9 -8 19 12 0 17
15 76 1 37 -5 -8 -9 -6 -12 25 -17 -13 -27 -34 0 -30 27 16 -3 -19 31 -37 30 33 28 -2 5 -21 15 35 -33 -10 32 -16 -25 13 9 8 22 23 7 10 -15 -1 -4 29 -32 -26 -38 20 11 12 -29 -22 -7 17 14 26 -35 -18 19 -28 18 6 21 -31 -23 -11 4 3 36 -24 34 -36 -14 -20 24 2
-3 40 -7 1 -20 -14 16 -5 -10 19 9 -11 13 8 -18 -9 0 12 -4 17 -6 -16 -12 -15 -2 18 11 14 6 10 -17 -13 7 3 5 15 -19 -8 -1 4 -3 2
-29 69 12 29 -31 -4 13 -21 33 18 19 -15 -18 17 34 -24 -30 -6 20 31 11 22 -32 -3 32 -9 -20 30 -23 25 6 21 -22 14 -34 -33 -17 -19 8 27 10 5 -16 -26 -1 -29 2 -7 7 26 -12 24 9 -14 3 -11 -5 15 28 -2 4 -25 -27 23 0 1 -13 -8 -10 -28 16
25 29 6 -11 1 -9 7 -5 14 4 -7 -13 -1 11 9 -14 0 10 -3 2 -2 13 -12 8 -4 -10 -6 -8 3 12 5
-2 24 2 1 -7 11 -5 10 6 -4 7 0 -2 -10 4 9 -3 -8 5 8 -1 3 -9 -11 -12 -6
87 90 -33 -27 9 -26 -22 -16 32 -13 -2 -23 3 34 21 -44 -20 -7 -43 -5 19 -45 1 -14 12 2 -17 8 -1 38 44 5 -25 41 -8 -30 23 11 -38 10 -42 -6 0 -37 6 28 -29 30 -39 36 -10 17 18 29 -32 26 -36 13 -4 -19 14 40 -11 16 22 -21 42 -40 20 15 33 35 -35 -31 -34 39 -24 -12 4 31 -15 -28 -3 27 37 25 43 -9 -41 -18 24 7
-1 22 -3 6 -8 9 -1 7 1 4 0 -5 -2 8 2 -4 3 -10 -7 -11 -6 5 10 -9
