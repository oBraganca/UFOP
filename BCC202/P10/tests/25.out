27 4
27 1
28 5
81 6
44 6
62 5
55 5
35 5
96 4
30 4
57 4
10 1
46 5
20 4
4 6
51 6
49 5
53 5
27 5
39 5
44 5
29 5
11 5
90 7
5 1
